metis-indexing/src/test/java/eu/europeana/indexing/tiers/view/RecordTierCalculationViewTest.java

```diff
@@ -54,4 +54,4 @@ void objectCreationTest() {
     assertNotNull(recordTierCalculationView.getMetadataTierBreakdown());
   }
 
-}
\ No newline at end of file
+}
```

Comment:
same here a test is missing with ```true``` to cover the new feature.

Reply:
Fixed it